nabs/visualizations.py

```diff
@@ -0,0 +1,56 @@
+import matplotlib.pyplot as plt
+from .plan_stubs import get_sample_targets
+
+
+def show_shot_targets(sample_name, path):
```

Comment:
Previously I was plotting the targets based on MxN values: 10 x 5 grid for example - and the problem here was that I needed to know if we used snake-like pattern or not. Now, I am plotting the actual xy points and this solves that but it would have been nice to also be able to do the MxN grid - is there a nice way to take those `xy` points in `matplotlib` and convert them to only their MxN index on the grid?
From this: (their x, y positions)
![true_target_grid](https://user-images.githubusercontent.com/62306310/106794187-52852e80-660d-11eb-9c3a-0f2bc37fb925.PNG)
To this: (their MxN values (M=rows, N=columns))
![show_grid_plot](https://user-images.githubusercontent.com/62306310/106794202-57e27900-660d-11eb-88e4-b6882396cdd2.PNG)


Please note these plots are using different data.

Reply:
I don't think there is a matplotlib built-in for "un-skewing" the lines here- you probably just have to track the columns yourself if you want to build something like this.